griptape/drivers/prompt/anthropic_prompt_driver.py

```diff
@@ -24,17 +25,20 @@ class AnthropicPromptDriver(BasePromptDriver):
     )
 
     def try_run(self, prompt_stack: PromptStack) -> TextArtifact:
-        prompt = self.prompt_stack_to_string(prompt_stack)
         response = anthropic.Anthropic(api_key=self.api_key).completions.create(
-            prompt=prompt,
-            stop_sequences=self.tokenizer.stop_sequences,
-            model=self.model,
-            max_tokens_to_sample=self.max_output_tokens(prompt),
-            temperature=self.temperature,
+            **self._base_params(prompt_stack)
         )
-
         return TextArtifact(value=response.completion)
 
+    def try_stream(self, prompt_stack: PromptStack) -> Iterator[TextArtifact]:
+        response = anthropic.Anthropic(api_key=self.api_key).completions.create(
+            **self._base_params(prompt_stack),
+            stream=True
```

Comment:
Explicitly passing `True` instead of `self.stream` for 2 reasons:
1. Anthropic's SDK has the type for `stream` as `Literal[True]`. Meaning that passing `self.stream` yields the following pyright error: `Argument of type "bool" cannot be assigned to parameter "stream" of type "Literal[True]" in function "create" "bool" cannot be assigned to type "Literal[True]`. Other LLM SDKs do not have this issue.
2. `self.stream` is really intended for `BasePromptDriver.run()` to determine whether to call `try_run` or `try_stream`. If we use it here too, we introduce the potential for a user setting `stream=False` in the Prompt Driver, but then calling `try_stream`. With this implementation, `try_stream` will always pass the correct streaming parameters to the LLM.

Let me know if I'm thinking about this wrong.